src/hooks/useUserData.ts

```
@@ -103,14 +106,15 @@ export const useApps = () => {
     : undefined;
   
-  // If we have initial data, use it directly without making network requests
   if (initialAppsArray && initialAppsArray.length > 0) {
     return { apps: initialAppsArray, appsError: null, appsLoading: false };
   }
 
-  // Only make network request if no initial data is available
+  const path = typeof window !== 'undefined' ? `${window.location.pathname}/apps` : '/apps';
+  const base = resolveBaseUrl();
+
   const { data, error, isLoading } = useSWRImmutable<AppClient[]>(
```

ESLint

[error] React Hook "useSWRImmutable" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)